doc/on_chain_spec.md

```diff
@@ -669,3 +669,175 @@ that distributing their respective portions of the auction fee
 can be done in a single transaction.
 Later on, this can be generalized in a straightforward way
 to accommodate incremental fee distribution to a larger number of delegates.
+
+# Extensions
+
+## Open Auctions
+
+This section describes an extenion that transforms the auctions from being closed to open.
+
+This means that, rather than having a fixed number of participants that must be decided beforehand,
+the number of pariticipants in the auction can dynamically grow,
+but is still gated by the sellers approval.
+
+To do this, we will require the seller to generate a signature
+for each bidder they want to allow in the auction.
+
+This signature can be generated by signing
+a pair consisting of the `hydraHeadId` and `bidderPKH`
+
+```haskell
+import Crypto.Sign.Ed25519
+
+generateSellerSignature :: SigningKey PaymentKey -> HeadId -> PubKeyHash -> BuiltinByteString
+generateSellerSignature skey headId bidderPKH = toBuiltin $ dsign skey $ sellerSignatureMessage headId bidderPKH 
+
+sellerSignatureMessage :: HeadId -> PubKeyHash -> BuiltinByteString
+sellerSignatureMessage headId bidderPKH = toByteString headId <> toBysteString bidderPKH
```

Comment:
Not `headID`. Auction ID (i.e. the asset class / currency symbol of the voucher token)